UKHO.D365CallbackDistributorStub.API/UKHO.D365CallbackDistributorStub.API/Models/Request/RecordCallbackRequest.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace UKHO.D365CallbackDistributorStub.API.Models.Request
+{
+    public class RecordCallbackRequest
+    {
+        public CallbackRequest? callBackRequest { get; set; }
+        public string? subscriptionId { get; set; }
+        public Guid guid { get; set; }
+    }
```

Comment:
Please fix naming rule violation issue for all 3 variables.

Reply:
Renamed